webapp/src/app/shared/multiline-zoom-graph/multiline-zoom-graph.component.ts

```diff
@@ -1377,27 +1383,54 @@ export class MultilineZoomGraphComponent implements OnInit, AfterViewInit {
       for (let i = 0; i < self.graphLinesData.length; i++) {
         
 
-        this.focus
-          .append("rect")
-          .attr("class", "hover rectBlueBox"+i)
-          .attr("fill", "#506EA7")
-          // .attr("fill-opacity", "0.9")
-          .attr("height", "14px")
-          .attr("width", "14px")
-          // .attr("display", "none")
-          // .attr("text-align", "middle")
-          // .attr("rx", 3)
-          // .attr("ry", 3)
-          .attr("x", 5)
-          .attr("dy", "0.50em");
-          // .attr("y", -7);
+        // this.focus
```

Comment:
please remove the commented code, if it is not usable

Reply:
I am working on that component. I will undergo few more changes. However, i will remove some not usable commented code and re-push.